src/summary/summary/prompt.py

```diff
@@ -0,0 +1,52 @@
+# ruff: noqa
+
+
+def get_instructions(transcript):
```

Comment:
find a way to modify this prompt without having to build a new docker image / release